src/Tests/chococcm.tests.ps1

```diff
@@ -1,26 +1,20 @@
+Describe "Module Import Checks" {
 
-
-Describe "Module Importability" {
-    
     BeforeAll {
-        $module = (Get-ChildItem "$($env:BuildRepositoryLocalPath)" -Recurse -Filter *.psd1).FullName[1]
-
+        $module = (Get-ChildItem "$PSScriptRoot\.." -Recurse -Filter *.psd1)[0].FullName
```

Comment:
This is the case already, we actually copy the test files to the output folder before testing during the build process, so when running the build it will be testing the built module in the output folder as well.

Though, as I may have mentioned, these tests are almost all testing next to nothing at present anyway, so it's a moot point until we get to fixing a bunch of these 😂